culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+contributor: the only requirements are that you have a change you need to
+propose, and that you agree to be a respectful participant in the review
+process. (When we have a code of conduct--link to it here.)
+
+### Reviewer
+
+The reviewers are the people who review the change, suggest changes to it, and
+eventually approve it. Anyone in the community can act as a reviewer: the only
+requirements are that you have some insight to share (or questions to ask)
+about the change, and that you agree to be a respectful participant in the
+process.
+
+### Maintainer
+
+The maintainers are responsible for the health and maintenance of the project,
+and have final say in the review process. Maintainers can block changes they
+think are harmful, or approve a change over others' objections. Someone can be
+a maintainer within a specific domain: for example, acting as maintainer for a
+particular repository, or even maintaining a particular guide. Since
+maintainers have special powers, they also have a special responsibility to act
+thoughtfully and fairly. New maintainers are chosen by the existing
+maintainers. In general, people who have a long track record of acting as
+contributors and reviewers may be good candidates for becoming maintainers.
+
+## Lifecycle of a Review
+
+### Proposal
+The first step is for a contributor to propose a _change_.
+The contributor needs to do two things at this stage:
+- Flesh out the change (e.g. writing the code), so that there is something to review
+- Explain the change (e.g. writing a pull request description), so that reviewers can easily understand it
+
+Both steps are vital, and contributors should put real effort into explaining the change well.
+Its not fair to ask a reviewer to puzzle through an undocumented change.
+
+Finally, before requesting a review, the contributor should _self-review_ their change.
+It's amazing how many issues can be spotted and fixed during self-review.
+This is respectful of reviewers' time, and makes the process a lot smoother.
+
+### Review
+
+Once a change is ready and has been self-reviewed, it's ready for review. Since
+most SourceCred artifacts are tracked via repositories on GitHub, this will
+usually mean creating a pull request.
+
+Once the change is proposed, anyone in the project can add their own review.
+The person proposing the change can also request specific reviewers, if certain
+individiuals have context or knowledge thet will make their reivew particularly
+valuable.
+
+On finishing their review, the reviewer can:
+- ask questions to better understand the change
+- suggest changes to it
+- approve the change
+
+Most changes go through a few rounds of reviews before being approved.
+
+### Approval and Merging
+
+During the review process, a successful change will accumulate "approvals" from
+satisfied reviewers. At some point, the change will have "enough approvals" and
+will be ready to merge. There's no hard-and-fast rule to determine when a
```

Comment:
While this makes sense for code changes, changes to official docs, I think relying on GitHub as a governance mechanism (as is done in Bitcoin, Ethereum, etc.) leads to problems of centralization when using it for more broad governance proposal. For instance, if someone proposes a change to a document artifact adding a new principle to a values statement, does the maintainer get a voto due to their monopoly on merge access? Something like this seems better decided with cred-weighted voting or other mechanism.

Reply:
Definitely share this view. While GitHub may have an open culture, from a technical perspective it uses authorities (the maintainers). To protect you from those authorities, you have this button
![Fork](https://user-images.githubusercontent.com/497556/72224681-5b780d80-357d-11ea-991c-646e89e9b3ab.png) <sup>One-click to be president of your own corner of the internet!</sup>

And this works very well for things that are **inexpensive to copy**. Like documentation and code.

But when something won't be easily copied:
- Like the people in a community
- Or the domain name it has
- Or the funding it received

That Fork button gives you _no protection at all_. Nice people might try to reduce the '[cost of forking](https://github.com/sfosc/sfosc/issues/5#issuecomment-485144478)'. But it's not when you agree with nice people that you need protections :smile:.

So using _only GitHub_ as your governance model, is the equivalent of giving a sitting president the power to veto the outcome of an election... There are better ways to do it.

---

That said. Reviewing, maintaining and governing are very different things so I think it's unrelated to "Review culture". So let's make a document in this repo to share some information about the SourceCred community does governance.

Reply:
>Let's make a document in this repo to share some information about how the SourceCred community does governance.

This is a good idea. Though the process is in enough flux right now I'm not sure it's worth the effort yet. Once we settle on some more formal governance mechanisms (e.g. get cred weighted voting going), and those are fairly stable, it makes sense to document them well. For now, linking to the [TBD post on Discourse](https://discourse.sourcecred.io/t/tbd-temporary-benevolent-dictator/376) post as we've been doing seems to suffice. Though as we build out user guides for newcomers, we should be on the lookout for opportunities to convey all of this stuff better.